### ISSUE TITLE
Gradient Picker onLastUpdate callback

### ISSUE THREAD

**Issue by nzbin** [easylogic/colorpicker#34]
**Is your feature request related to a problem? Please describe.**
The `onLastUpdate` callback doesn't working on Gradient Picker.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by easylogic** [comment]
hi @nzbin  

The answer was late. I'll check and let you know.

**Comment by easylogic** [comment]
i released v1.10.8. 

please check below code 

```js
var picker = new EasyLogicColorPicker.createGradientPicker({
    position: 'inline',
    container: item,
    type: type,
    colorpickerOptions: {
        type: 'chromedevtool'
    },
    gradient: 'linear-gradient(to right, red 0%, white 100%)',
    onChange : function (gradientString, gradientObject) {
        // console.log('change', gradientString, gradientObject);
    },
    onLastUpdate: function (gradientString, gradientObject) {
        console.log('on last changed', gradientString, gradientObject);
    }
});

picker.setValue('radial-gradient(circle, yellow 10%, black 100%)')
```
